pootle/settings/50-project.conf

```diff
@@ -92,6 +92,7 @@ INSTALLED_APPS = [
     'pootle_profile',
     'pootle_statistics',
     'pootle_terminology',
+    'pootle_fs',
```

Comment:
Are you expecting that pootle_fs will be immediately used in real production environment by the majority of customers? If not, does it make sense not to enable it in the config by default, and keep it as an "experimental" feature for a while? Those who will be willing to use it will explicitly but I guess it would be safer to keep the core as stable as possible.

I think we need to take this approach in general — new pluggable features are released as "opt-in" by default, and then, when the code is considered stable and used by early adopters, it can become "opt-out" if necessary.